USING_DOCKER.md

```diff
@@ -40,26 +40,17 @@ cd <top-level-PRQL-directory>
 docker build -t prql .
 ```
 
-_Optional:_ The `pre-commit` program makes a quick check
-for silly mistakes of all files: rust formatting,
-`.yaml` problems, Markdown style, pretty-printing, etc.
-Run it on the machine that hosts your Docker container.
+_Optional:_ Install `pre-commit` on the machine that hosts Docker.
+It runs several
+[Static Checks](./DEVELOPMENT.md#tests) to ensure code consistency.
 You can also configure `git` to run `pre-commit` automatically
-for each commit. [^1]
+for each commit with the second (one-time) command below.
 
 ```bash
-pre-commit run -a
+pre-commit run -a                # Run checks manually
+pre-commit install install-hooks # (one time) install the git hooks
```

Comment:
I thin this needs to be `--install-hooks`; the current command raises for me

(I can't seem to make a suggestion in GH after merge)

Reply:
Weird, as you say, the real code is correct